libraries/core_libs/consensus/include/vote_manager/vote_manager.hpp

```diff
@@ -280,7 +283,8 @@ class VoteManager {
   blk_hash_t reward_votes_block_hash_;
   PbftRound reward_votes_period_;
   PbftRound reward_votes_round_;
-  mutable std::shared_mutex reward_votes_info_mutex_;
+  // reward_votes_ will only keep additional reward votes which are not part of the original 2t+1 cert votes
```

Comment:
lets call it extra_reward_votes or something similar

Reply:
Fixed